### ISSUE TITLE
Bug after in link after <ul>

### ISSUE THREAD

**Issue by jonpither** [yogthos/markdown-clj#80]
````
(md/md-to-html-string "* hi

[a link](https://see-here)")
````
->

````
<ul><li>hi</li></ul><p>þǫ0þǫ</p>
````

Strange..

**Comment by yogthos** [comment]
ok should be fixed in the latest `0.9.82`, let me know if everything looks good

**Comment by jonpither** [comment]
Fixed in 0.9.82 - thanks :-)
